--- ./bin/DVDAuthorWizard-Builder.sh.orig	2017-03-18 22:18:16.000000000 +0100
+++ ./bin/DVDAuthorWizard-Builder.sh	2017-10-02 09:22:00.955789419 +0200
@@ -327,13 +327,13 @@
  # -x			autodetect video, ignore audio
  # -I 5			Deinterlace with linear blend (lb)
  # -Z			Convert frame size
- # --keep_asr		Fit input image in target framesize without changing aspect
+ # EOL --keep_asr		Fit input image in target framesize without changing aspect
  # -y im,null		Export video to ImageMagick, ignore audio
 
  DiskSpaceWarning "$(pwd)" 100
 
  transcode -i "$1" -x auto,null -I 5 $TranscodeEffectString -Z $2 \
-           --keep_asr -y im,null -F png -c $3-$4 -o frame%d.png 2>&1
+           -y im,null -F png -c $3-$4 -o frame%d.png 2>&1
 
  DiskSpaceError "$(pwd)"
 
@@ -1069,7 +1069,7 @@
 
    pushd "$WorkingDir"
 
-   dd if=/dev/zero bs=2 count=$NumAudioFrames | sox -r 48000 -t .sw - -t .wav -w /dev/stdout \
+   dd if=/dev/zero bs=2 count=$NumAudioFrames | sox -r 48000 -t .sw - -t .wav /dev/stdout \
                                               | mp2enc -b 128 -r 48000 -o soundtrack.mp2 < /dev/stdin
 
    popd
@@ -1093,12 +1093,12 @@
 
    if [ "$DetectedSampleRate" == 48000 ]; then
 
-     sox "$SoundtrackFile" -t .wav -w "$WorkingDir/soundtrack.wav" \
+     sox "$SoundtrackFile" -t .wav "$WorkingDir/soundtrack.wav" \
          fade t 0 $Length  $FadeoutLength trim 0 $Length
 
    else
 
-     sox "$SoundtrackFile" -t .wav -w -r 48000 "$WorkingDir/soundtrack.wav" \
+     sox "$SoundtrackFile" -t .wav -r 48000 "$WorkingDir/soundtrack.wav" \
          fade t 0 $Length  $FadeoutLength trim 0 $Length resample -qs
 
    fi
